src/components/App.js

```diff
@@ -1,9 +1,12 @@
 import './App.css';
+import BooksForm from './BooksForm';
+import BooksList from './BooksList';
 
 function App() {
   return (
     <div className="App">
-      <h1>Hello!</h1>
+      <BooksForm />
+      <BooksList />
```

Comment:
### [Optional]
You could use react fragment here, just the way you used it in `src/components/BooksForm.js`